workers/processor.nomad.tpl

```diff
@@ -44,6 +44,8 @@ job "PROCESSOR" {
         RAW_PREFIX = "${{RAW_PREFIX}}"
         TEMP_PREFIX = "${{RAW_PREFIX}}"
         PROCESSED_PREFIX = "${{PROCESSED_PREFIX}}"
+
+        NOMAD_HOST = "${{NOMAD_HOST}}"
```

Comment:
This is probably unnecessary right? The downloaders template file doesn't have it and it's able to communicate with Nomad.